openff/utilities/testing.py

```diff
@@ -43,14 +43,12 @@ def skip_if_missing_exec(exec: Union[str, list[str]]) -> "MarkDecorator":
     elif isinstance(exec, list):
         execs = exec
     else:
-        raise ValueError(
-            "Bad type passed to skip_if_missing_exec. " f"Found type {type(exec)}"
-        )
+        raise ValueError("Bad type passed to skip_if_missing_exec. " f"Found type {type(exec)}")
 
     found_exec = False
     for exec_ in execs:
         found_exec = found_exec or has_executable(exec_)
 
-    reason = f"Package {str(exec)} is required, but was not found."
+    reason = f"Package {exec!s} is required, but was not found."
```

Comment:
(not blocking) both of these seem weird/worse to me. Maybe a weird interaction with a formatting tool? Not blocking either way, just stood out.

Reply:
It's probably an unnecessary cast tbh, but it's not going to be called at scale so the runtime penalty is small. I'm going to leave it as-is until it presents an issue

https://docs.astral.sh/ruff/rules/explicit-f-string-type-conversion/#explicit-f-string-type-conversion-ruf010